test/node/ol/proj.test.js

```diff
@@ -158,7 +158,7 @@ describe('ol/proj.js', function () {
   describe('fromUserResolution()', function () {
     it("adjusts a resolution for the user projection's units", function () {
       useGeographic();
-      const user = 1 / METERS_PER_UNIT.degrees;
+      const user = 1 / ((Math.PI * 6378137) / 180);
```

Comment:
It would be better to use `getProjection('EPSG:4326).getMetersPerUnit()` here.

Reply:
Ok, thanks!